Cargo.toml

```diff
@@ -129,6 +129,7 @@ tokio-uds = { version="0.2", optional = true }
 [dev-dependencies]
 env_logger = "0.6"
 serde_derive = "1.0"
+tempfile = "3.0"
```

Comment:
It would be great if we could avoid using extra dependencies

Mostly because actix-web takes long time to build tests already 😄

Reply:
I can also do it with a regular `std::fs::File`. The only problem would be that in some obscure corner cases, it will leave a in the user's temp directory. I guess that is an acceptable tradeoff?

Reply:
Yeah it is fine, I'd like to not increase compile time as much as possible.

Well, to avoid creating empty files, just tag all examples as `no_run` so that `cargo test` would only compile them